wordpress-mobile/AztecEditor-iOS
# Chinese H1 Input: Running only upon replacement

### Details:
In this PR we're updating the workaround introduced via #814:

- It'll be executed only when needed: upon text replacement
- We're switching to `fixAttributes` (vs `invalidateAttributes`)
- Added new unit tests!

Ref. #811

Needs Review: @diegoreymendez 

### To test:
1. Verify that the unit tests are green
2. Launch the empty editor
3. Insert a newline
4. Switch to the **Chinese Simplified Pinyin** keyboard
5. Type `s` and tap over any of the replacement suggestions
6. Repeat the step above. 

Verify that the keyboard suggestion selected in (6) remains onscreen.
